MapboxNavigation/RouteMapViewController.swift

```diff
@@ -976,9 +980,19 @@ protocol RouteMapViewControllerDelegate: class {
     func mapViewControllerDidOpenFeedback(_ mapViewController: RouteMapViewController)
     func mapViewControllerDidCancelFeedback(_ mapViewController: RouteMapViewController)
     func mapViewControllerDidCancelNavigation(_ mapViewController: RouteMapViewController)
+    func mapViewController(_ mapViewController: RouteMapViewController, shouldShowCancelButtonIn bottomBanner: BottomBannerView) -> Bool
     func mapViewController(_ mapViewController: RouteMapViewController, didSend feedbackId: String, feedbackType: FeedbackType)
     
     func mapViewController(_ mapViewController: RouteMapViewController, mapViewUserAnchorPoint mapView: NavigationMapView) -> CGPoint?
     
     func mapViewControllerShouldAnnotateSpokenInstructions(_ routeMapViewController: RouteMapViewController) -> Bool
 }
+
+extension RouteMapViewControllerDelegate where Self: UIViewController {
```

Comment:
Note that Objective-C doesn’t support generics on protocols, nor default method implementations in protocols. These limitations may become relevant if we decide to make RouteMapViewController public.